### PR TITLE
Minor refactor

### PR DESCRIPTION
**Pull request by BMartinos**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,8 +15,15 @@
...
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "browserify --entry src/index.js | uglifyjs --output dist/bundle.js",
+    "dev": "watch 'npm run build' src",
+    "test": "npm run -s test:code && npm run -s test:style",
+    "test:code": "tap -R nyan --cov test/**/*.js",
+    "test:these-files": "tap --reporter=spec",
+    "test:style": "standard --verbose | snazzy && htmlhint app index.html",
+    "test:debug": "tap --debug -t 0 test/**/*.js",
+    "test:these": "tap -R nyan",
```
Comment (rcrichton) [inline package.json]: test:these and test:these-files do pretty much the same thing just with a different reporter. I'd say keep `test:these` as I like that syntax better.

File: src/validators/idNumber.js [hunk 2]
```diff
@@ -1,36 +1,82 @@
...
+    restrict: 'A',
+    require: 'ngModel',
     scope: {
       checkIdNumber: '=checkIdNumber'
     },
-    link: function(scope, element, attributes, ngModel) {
-
-      scope.$watch('checkIdNumber', function(newValue, oldValue) {
+    link: function (scope, element, attributes, ngModel) {
+      scope.$watch('checkIdNumber', function (newValue, oldValue) {
```
Comment (rcrichton) [inline src/validators/idNumber.js]: Maybe this should be `checkRSAIdNumber` as it's very specific to RSA IDs? Are we able to write more validators and plug them in easily for other IDs?

Comment (rcrichton) [reply src/validators/idNumber.js]: I see there it still the SA validator below. I'm not sure I understand the difference between the two. Could you explain?

File: src/validators/south-african-id-number-validator.js [hunk 3]
```diff
@@ -44,4 +43,4 @@ var validate = function(idNumber) {
   return identityNumberValidatorResult
 }
 
-exports.validate = validate
\ No newline at end of file
+exports.validate = validate
```
Comment (rcrichton) [inline src/validators/south-african-id-number-validator.js]: Can this not be deleted? It seems very similar to `idNumber.js`.

File: test/test.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+const tap = require('tap')
+
+tap.test('Test: should run a fake test and pass', (t) => {
+  t.end()
+})
```
Comment (rcrichton) [inline test/test.js]: Maybe remove this now that you have actual test.

File: src/validators/idNumber.js (hunk 2 above)
Comment (BMartinos) [reply src/validators/idNumber.js]: The file can be removed as this was the old location for the RSA validate function.

File: README.md [hunk 5]
```diff
@@ -395,3 +397,38 @@ Each section contains rows and each row has fields. See the example of a section
...
+    "action": "showhide" // action to perform on the field ("disabled", "required", "showhide")
+  }]
+}
+```
+
+Below is a skip logic fucntion checks that will execute a function called "calculateBMI" when both the "weight" and "height" fields have values. for a field called "bmi" and shows the field if the value is bigger than 0
+```
+"skipLogic": {
+  "func": {
+    "execute": "placeholderForCalculateBMI", // function to execute. !NB this must be a function attached to this property to execute successfully. The function will receive 1 params (params)
```
Comment (rcrichton) [inline README.md]: "The function will receive 1 params (params)" - this seems weird, not sure what you meant to write here.

**Pull request merged**